compiler/aarch64/codegen/BinaryEvaluator.cpp

```diff
@@ -703,6 +703,144 @@ OMR::ARM64::TreeEvaluator::vmulEvaluator(TR::Node *node, TR::CodeGenerator *cg)
    return inlineVectorBinaryOp(node, cg, mulOp);
    }
 
+TR::Register *
+OMR::ARM64::TreeEvaluator::vdivIntHelper(TR::Node *node, TR::CodeGenerator *cg)
+   {
+   struct DivOps {
```

Comment:
Please use the following indentation style:

```
   struct DivOps
      {
      TR::InstOpCode::Mnemonic vectorToGPROp;
      ...
      }
```

Reply:
Fixed the indentation.